### ISSUE TITLE
Failed Codes

### ISSUE THREAD

**Issue by ItsLeon15** [Bonnie39/tominecon_cracker#4]
I was wondering if there could be a discussions tab to discuss and plan who does which numbers. I've already done between 0-2,700,000 in the last day but as the current program starts from 0 it means that other people would be doing the same tries which is a bit wasteful. Making sure people know which numbers have already been done would save a lot of time.

**Comment by ItsLeon15** [comment]
Yeah sure, I'm down for that. Not sure how to create a group chat so someone else would have to.

**Comment by sqlskid** [comment]
https://t.me/+mXcLjOyh7FdiMjFk
you can try joining

**Comment by oong819** [comment]
or post your tries on Reddit r/tominecon so it's public and more accessible

**Comment by oong819** [comment]
wait when you say 0 you mean "0" or "0000000000000000"?

**Comment by ItsLeon15** [comment]
> r/tominecon

Yeah, I meant 0000000000000000, sorry.

**Comment by dandanthedev** [comment]
Heyo, maybe its an idea to build some kind of "tracker" that will asign numbers to people?

**Comment by dandanthedev** [comment]
I could help with implementing that

**Comment by oong819** [comment]
According to https://www.reddit.com/r/tominecon/comments/1cq3bp1/why_the_password_likely_isnt_16_digits/ the claim that the password is 16 digits are just joke
